src/space.js

```diff
@@ -169,23 +125,16 @@ export default class ColorSpace {
 		}
 
 		if (ret) {
-			ret = this.#processFormat(ret);
+			ret = processFormat(ret, this);
 			return ret;
 		}
 
 		return null;
 	}
 
-	#path;
-
-	#getPath () {
-		let ret = [this];
-
-		for (let space = this; space = space.base;) {
-			ret.push(space);
-		}
-
-		return ret;
+	// We cannot rely on simple === because then ColorSpace objects cannot be proxied
+	equals (space) {
+		return this === space || this.id === space.id;
```

Comment:
shouldn't this logic be used in the `to()` method then to replace the `this === space` check ?

Reply:
It …is? Line 162. Am I missing another place?

Reply:
Ah, thanks. Should be fixed now.